en_US/data-integration/data-bridge-kafka.md

```diff
@@ -274,24 +274,64 @@ mqttx pub -i emqx_c -t t/1 -m '{ "msg": "Hello Kafka" }'
+
+### Create a Rule SQL
+
+1. Go to EMQX Dashboard, and click **Integration** -> **Rules**.
+
+2. Click **Create** on the top right corner of the page.
+
+3. Enter a rule ID, for example, `my_rule`.
+
+4. Enter the following statement in the **SQL Editor** if you want to forward the messages transformed from the Kafka source`$bridges/kafka_consumer:<sourceName>` to EMQX.
+
+   Note: If you want to specify your own SQL syntax, make sure that the `SELECT` part includes all fields required by the republishing action set in later steps.
+
+   ```sql
+   SELECT
+     *
+   FROM
+     "$bridges/kafka_consumer:<sourceName>"
+   ```
+
+   Note: If you are a beginner user, you can click **SQL Examples** and **Enable Test** to learn and test the SQL rule.
+
+### Add Kafka Consumer Source as Data Input
 
-6. **Key Encoding Mode** and **Value Encoding Mode**: Select the encoding mode for Kafka message key and message value.
+1. Select the **Data Inputs** tab on the right side of the Create Rule page and click **Add Input**. 
 
-7. Click **Add** to add at least one Kafka-to-MQTT topic mapping in the **Topic Mapping** field. For example, enter `testtopic-out` in **Kafka Topic** and `t/1` in **MQTT Topic** for this demonstration. The **MQTT Payload Template** subfield specifies the MQTT payload that should be used, and has the following Kafka message fields available for templating:
+2. Select **Kafka Consumer** from the **Input Type** dropdown list. keep the **Source** dropdown box to the default `Create Source` option, or choose a previously created Kafka Consumer source from the **Source** dropdown box. This demonstration creates a new consumer source and adds it to the rule.
+
+3. Enter the name and description of the Source in the corresponding text boxes below.
+
+4. In the **Connector** dropdown box, select the `my-kafka-consumer` connector you just created. You can also click the button next to the dropdown box to quickly create a new connector in the pop-up box, with the required configuration parameters referring to [Create a Kafka Consumer Connector](#create-a-kafka-consumer-connector).
+
+5. Configure the following fields:
+
+   - **Kafka Topic**: Specify the Kafka topic from which the consumer source will subscribe to for receiving messages.
+   - **Group ID**: Specify the consumer group identifier for this source. If not provided, a group ID will be automatically generated based on the source name.
+   - **Key Encoding Mode** and **Value Encoding Mode**: Select the encoding mode for Kafka message key and message value.
+
+6. Click **Add** to add at least one Kafka-to-MQTT topic mapping in the **Topic Mapping** field. For example, enter `testtopic-out` in **Kafka Topic** and `t/1` in **MQTT Topic** for this demonstration. The **MQTT Payload Template** subfield specifies the MQTT payload that should be used, and has the following Kafka message fields available for templating:
```

Comment:
Please check if the "Topic Mapping" configurations should be removed, since this field is not in Dashboard now. @id @zmstone